client/src/www/app/outline_server_repository/server.fake.ts

```diff
@@ -34,7 +34,7 @@ export class FakeTunnel implements PlatformTunnel {
     return hostname === FAKE_UNREACHABLE_HOSTNAME;
   }
 
-  async start(config: ShadowsocksSessionConfig): Promise<void> {
+  async start(_name: string, config: ShadowsocksSessionConfig): Promise<void> {
```

Comment:
*nit* remove underscore? Or maybe rename it to `unusedName`?

```suggestion
  async start(name: string, config: ShadowsocksSessionConfig): Promise<void> {
```

Reply:
The linter complains if I don't prefix unused variables with a _.